src/main/java/no/nav/veilarbaktivitet/avtaltMedNav/ForhaandsorienteringDAO.java

```diff
@@ -97,6 +96,14 @@ public Forhaandsorientering getById(String id) {
         }
     }
 
+    public List<Forhaandsorientering> getById(List<String> ider) {
+        String idString = ider.stream().collect(Collectors.joining("','", "'","'"));
+        //language=sql
+        String sql = String.format("SELECT * FROM FORHAANDSORIENTERING WHERE id IN(%s)",idString);
+
+        return database.query(sql, ForhaandsorienteringDAO::map);
```

Comment:
Jeg synes det er uheldig med denne Database-wrapperen, for det hindrer oss i å bruke det rike JdbcTemplate apiet til spring. Det hadde vært bedre å brukt JdbcTemplate direkte. Og så hadde det vært bedre å migrert over til å bruke NamedParameterJdbcTemplate. For dette tilfellet spesielt, kunne du da brukt id-listen som et parameter
`namedParameterTemplate.query("select * from forhaandsoriantering where id in (:ider), parameter, ForhaandsorienderingDAO::map)`